src/Repositories/ItemRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DomainModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Repositories
+{
+    public class ItemRepository : IItemRepository
+    {
+        private readonly DataContext _dataContext;
+
+        public ItemRepository(DataContext dataContext)
+        {
+            _dataContext = dataContext ?? throw new ArgumentNullException(nameof(dataContext));
+        }
+
+        public IQueryable<Item> All()
+        {
+            return _dataContext.Items.AsQueryable();
+        }
+
+        public void Save() { }
+
+        public void Delete() { }
```

Comment:
please implement the method. DataContext.Items is where the item should be deleted from.

Reply:
Implemented